src/lib/modules/aeris-catalog-layouts/aeris-search-criteria-layout/components/aeris-thesaurus-item-checkbox-layout.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <article v-if="isVisible" :style="applyTheme" class="aeris-thesaurus-item-checkbox-layout">
+    <section>
+      <input
+        :id="`${name}${getThesaurusItem.name}${getThesaurusItem.search}`"
+        :checked="getIsChecked"
+        type="checkbox"
+        @click="check()"
+      />
+      <label :for="`${name}${getThesaurusItem.name}${getThesaurusItem.search}`">{{ getThesaurusItem.label }}</label>
+      <aside v-if="sublevelLength > 0">
+        <span class="badge">{{ sublevelLength }}</span>
+        <i v-show="getThesaurusItem.deployed" class="far fa-minus-square deployed" @click="toggleLevel()" />
+        <i v-show="!getThesaurusItem.deployed" class="far fa-plus-square deployed" @click="toggleLevel()" />
+      </aside>
+    </section>
+
+    <article
+      v-show="getThesaurusItem.deployed && getThesaurusItem.thesaurusItems && getThesaurusItem.thesaurusItems.length"
+      class="thesaurusItemChild"
+    >
+      <aeris-thesaurus-item-checkbox-layout
+        v-for="thesaurusItemChild of getThesaurusItem.thesaurusItems"
+        :key="thesaurusItemChild.search"
+        :thesaurus-item="thesaurusItemChild"
+        :is-checked="getThesaurusItem.checked"
+        :thesaurus-type="thesaurusType"
+      ></aeris-thesaurus-item-checkbox-layout>
+    </article>
+  </article>
+</template>
+
+<script>
+export default {
+  name: "aeris-thesaurus-item-checkbox-layout",
+
+  props: {
+    name: {
+      type: String,
+      default: ""
+    },
+    theme: {
+      type: Object,
+      default: () => {}
+    },
+    thesaurusItem: {
+      type: Object,
+      default: () => {}
+    },
+    thesaurusType: {
+      type: String,
+      default: ""
+    },
+    isChecked: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  computed: {
+    isVisible() {
+      return this.thesaurusItem && this.thesaurusItem.name && this.thesaurusType ? true : false;
```

Comment:
si tu veux juste retourner true ou false tu peux juste écrire : "this.thesaurusItem && this.thesaurusItem.name && this.thesaurusType" sans l'opérateur ternaire

Reply:
tu as raison :-D